tools/pipelines/templates/include-test-perf-benchmarks.yml

```diff
@@ -0,0 +1,110 @@
+# Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+# Licensed under the MIT License.
+
+parameters:
+- name: poolBuild
+  type: object
+  default: Large
+
+- name: customSteps
+  type: stepList
+
+- name: testWorkspace
+  type: string
+
+- name: artifactPipeline
+  type: string
+
+jobs:
+  - job: runTests
+    displayName: Run tests
+    pool: ${{ parameters.poolBuild }}
+    variables:
+      feed: https://pkgs.dev.azure.com/fluidframework/internal/_packaging/dev/npm/registry/
+
+    steps:
+    # Setup. Need to checkout the repo in order to run @fluid-tools/telemetry-generator which we don't publish right now.
```

Comment:
out of curiosity--is longer-term plan to publish this and convert this pipeline to not install/build?

Reply:
No well-defined plan for that tool yet, but I think that's a good idea. Probably a well fit for `fluid-internal`